portal-web/docroot/html/taglib/ui/input_permissions/page.jsp

```diff
@@ -172,7 +172,7 @@ String modelName = (String)request.getAttribute("liferay-ui:input-permissions:mo
 				var inputPermissionsOptionsButton = document.getElementById('<%= uniqueNamespace %>inputPermissionsOptionsButton');
 
 				if (inputPermissionsOptionsButton) {
-					inputPermissionsOptionsButton.innerText = force ? '<%= LanguageUtil.get(request, "hide-options") %>' :'<%= LanguageUtil.get(request, "more-options") %>' ;
+					inputPermissionsOptionsButton.innerText = force ? '<%= StringUtil.replace(LanguageUtil.get(request, "hide-options"), "\'", "\\'") %>' : '<%= StringUtil.replace(LanguageUtil.get(request, "more-options"), "\'", "\\'") %>' ;
```

Comment:
Shouldn't `HtmlUtil.escapeJs` be used here?

Reply:
You are right, that works as well. I've updated the pull. Thanks!